### PR TITLE
updates to GDC and WGS

### PR DESCRIPTION
**Pull request by ekiernan**
Updates to the GDC Overview and the WGS methods doc

### CODE REVIEW

File: pipelines/broad/dna_seq/somatic/single_sample/wgs/gdc_genome/README.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
-# Genomic Data Commons (GDC) Whole Genome Somatic Single Sample Overview
-
-| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
-| :----: | :---: | :----: | :--------------: |
-| [GDCWholeGenomeSomaticSingleSample_v1.0.0](https://github.com/broadinstitute/warp/releases) | January, 2021 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in warp or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
+## Announcing a new site for WARP documentation!
 
+GDC Whole Genome Somatic Single Sample documentation has moved! Read more about the [Illumina Genotyping Array Pipeline](https://broadinstitute.github.io/warp/documentation/Pipelines/Genomic_Data_Commons_Whole_Genome_Somatic/) on the new [WARP documentation site](https://broadinstitute.github.io/warp/)!
```
Comment (nikellepetrillo) [inline pipelines/broad/dna_seq/somatic/single_sample/wgs/gdc_genome/README.md:3]: Should this say something like "Read more about the [GDC Whole Genome Somatic Single Sample Pipeline]" ?

**Review by nikellepetrillo — APPROVED** [review]
Looks good to me! Just one small comment.

File: pipelines/broad/dna_seq/somatic/single_sample/wgs/gdc_genome/README.md (hunk 1 above)
Comment (ekiernan) [reply pipelines/broad/dna_seq/somatic/single_sample/wgs/gdc_genome/README.md]: Absolutely! Thank you for catching!

**Review by khajoue2 — APPROVED** [review]
lgtm

**Pull request merged**